test/unit/views/payment-sheet-views/card-view.js

```diff
@@ -660,6 +660,37 @@ describe('CardView', function () {
         });
       });
     });
+
+    it('allows overriding styles options with class name for hosted fields', function () {
```

Comment:
FUN FACT jsdom is _very bad_ at `getComputedStyle` so we have that to look forward to if we ever change testing loaders